src/CoreBundle/Bridge/Chameleon/Module/NavigationTreeSingleSelect/NavigationTreeSingleSelect.php

```diff
@@ -86,15 +160,103 @@ public function Accept(\IMapperVisitorRestricted $visitor, $cachingEnabled, \IMa
             $rootNode = new \TdbCmsTree();
             $rootNode->SetLanguage(\TdbCmsUser::GetActiveUser()->GetCurrentEditLanguageID());
             $rootNode->Load($rootTreeId);
+            $visitor->SetMappedValue('breadcrumbStorageHTML', $this->createBreadcrumbStorage($rootNode));
+        }
+
+        $visitor->SetMappedValue('fieldName', $this->fieldName);
+
+        $url = $this->urlUtil->getArrayAsUrl(
```

Comment:
The variable could have directly the mapped name below ($treeNodesAjaxUrl; also so it is not overwritten for a different purpose below).

Reply:
Ok, I changed it. (This was copied/adapted from NavigationTree.php. It is the same there.)